pkg/runtime/runtime.go

```diff
@@ -1873,6 +1886,11 @@ func (a *DaprRuntime) Publish(req *pubsub.PublishRequest) error {
 	}
 
 	policy := a.resiliency.ComponentOutboundPolicy(a.ctx, req.PubsubName, resiliency.Pubsub)
+
+	if a.pubSubs[req.PubsubName].namespaceScoped {
```

Comment:
Please use `if ps.namespaceScoped {`, this map's read is under RLock.

Reply:
Done